### PR TITLE
Support Danish (DA)

### LINKED ISSUE #1948: Support Danish (DA)
- Wiktionary page: https://da.wiktionary.org/
- Model link: https://da.wiktionary.org/wiki/hygge (I wasn't entirely sure what you meant by "model link", but I hope you meant an example.)

It would be nice to add a template for Danish!

### PR DESCRIPTION
**Pull request by jolars**
Fixes #1948

Here is an attempt to support the Danish wiktionary (<https://da.wiktionary.org>). It's mostly working okay, but I have struggled to get variant support to work. Here's an example for "hunden", which is an inflection of "hund":

```
{{-da-}}
{{-noun-|da}}
{{infl|da|substantiv form}}
#Bestemt ental af '''[[hund]]'''.
```

I'm not sure how to deal with this since there's no template connected to the variant ("hund"). Maybe do some replacement through https://github.com/BoboTiG/ebook-reader-dict/blob/2aedd0a7d422bbac2b1fb327f93a13b754ee797e/wikidict/render.py#L299? Do you have any ideas?

### CODE REVIEW

**Comment by BoboTiG** [comment]
This is great, thanks @jolars! 🍾 

First, I did not check the code yet.
Second, lets tackle more complex variants later. We still have issues with some other locales because of the exact same lack of templates.

I'll get back to you soon (™️).

File: wikidict/find_templates.py [hunk 1]
```diff
@@ -20,6 +20,9 @@ def find_templates(in_words: Dict[str, str], locale: str) -> None:
     templates: Dict[str, str] = {}
     locale_sections = sections[locale]
     for in_word, code in in_words.items():
+        if locale == "da":
```
Comment (jolars) [inline wikidict/find_templates.py:23]: By the way, I'm not sure if this is really kosher, but `--find-templates` needed this to be able to correctly parse sections.

**Comment by BoboTiG** [comment]
Thanks a lot @jolars 🍾 

Do you think you could translate https://github.com/BoboTiG/ebook-reader-dict/tree/master/docs/da? For screenshots, you can ismply comment out them, I'll see to take them later if you have interesting words.

**Comment by BoboTiG** [comment]
@all-contributors please add @jolars for code

**Comment by jolars** [comment]
Sure, I can give it a shot, but I'm not a native speaker so it might not 
be perfect :)

**Comment by BoboTiG** [comment]
> Sure, I can give it a shot, but I'm not a native speaker so it might not be perfect :)

Something not perfect will be way enough. With time, maybe someone native will spot issues and fix them :)

**Pull request merged**